### ISSUE TITLE
club clothes ideas

### ISSUE THREAD

**Issue by ianklatzco** [ddrillini/uiuc-wendy#73]
DDRIllumni shirts
break away pants
jackets

**Comment by andrewipark** [comment]
Slight hijacking, but: do we have new DDRIllini branding?

**Comment by ianklatzco** [comment]
the logo currently on the github org/discord, but that's it.

**Comment by ianklatzco** [comment]
dropped a line in #create about a shirt design contest.

**Comment by ianklatzco** [comment]
shirts:
 * we want to do water-based ink (lasts a looooong time for 10% more expensive)
 * weiskamp screen printing in champaign
 * threadbird in chicago (brandon uses them for tournament shirts)

**Comment by emiliakedainis** [comment]
Weiskamp:

- 10% increase from regular cost to use water-based ink
- basic shirt quantity 25-49 costs $7.45 per shirt
- nicer shirt quantity 25-49 costs $8.43 per shirt
- staff favorite quantity 25-49 costs $9.12 per shirt
- no additional cost to change shirt color
- price on a per-color increase on the design, but designs have to be the same for each shirt to get mass production discount (e.g. one color on the design for basic shirts wound be around $7.45, but would increase to around $12 if we have three colors)


Threadbird:

- 35 cents per location per shirt to used water-based ink
- if you switch shirt colors within shirts, it doesn't cost extra, but you need to order at least 12 of each kind, but any breakdown of sizes
- amount of colors on design change price, but it's by 2 (e.g. 1 and 2 colors cost the same, 3 and 4 colors cost the same) 
- lots of shirt options (over 100 t-shirt options), best to look at website and choose shirt first 
- generally costs around $10 per shirt

**Comment by ianklatzco** [comment]
seems like weiskamp would offer better color options, and threadbird would lock us to ~1-2 colors.

with weiskamp, would everyone get to pick whatever color they wanted?

**Comment by emiliakedainis** [comment]
Yes. You can have as many shirt colors as you want for no additional cost, as long as the design color stays the same. Changing design colors is more expensive (e.g. a white shirt and a black shirt with a gold arrow don’t cost any different, but it would be more expensive to have a black shirt with a white arrow and a white shirt with a black arrow). There is no minimum quantity for shirt colors. You could have one white shirt if you choose to and they’d print it for no additional cost, assuming the design color is the same

**Comment by ianklatzco** [comment]
Seems like the better option for our needs. The one concern I have is dark / light differences — might cause the whole order to cost more, but it might be a nominal amount.

I guess, we should finalize the design and make an order form that asks for size, quantity, color. Starting a discussion in #create. @emiliakedainis, could you make the order form?

**Comment by andrewipark** [comment]
what ever became of this? We have a graduating class in a few months.

**Comment by ianklatzco** [comment]
other tasks higher priority. currently thinking about nice jackets (a la hぁck illinois) w/ embroidered "ddrillini" on the breast. guessing ~$40.
